container/app/entrypoint.py

```diff
@@ -0,0 +1,70 @@
+#!/venv/bin/python3
+#
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import argparse
+import json
+import os
+import subprocess
+
+from ovirt_imageio.client import _utils
+from ovirt_imageio._internal import qemu_img
+from ovirt_imageio._internal import qemu_nbd
+from ovirt_imageio._internal import nbd
+
+TICKET_PATH = "/app/ticket.json"
+OVIRT_IMAGEIO = "/venv/bin/ovirt-imageio"
+
+
+def _generate_ticket(uuid, socket, size):
+    ticket_info = {}
+    ticket_info["uuid"] = uuid
+    ticket_info["url"] = f"nbd:unix:{socket}"
+    ticket_info["size"] = size
+    ticket_info["timeout"] = 3000
+    ticket_info["inactivity_timeout"] = 300
+    ticket_info["ops"] = ["read"]
+    return ticket_info
+
+
+def create_ticket(ticket_id, image, socket):
+    image_size = qemu_img.info(image)["virtual-size"]
+    ticket_info = _generate_ticket(ticket_id, socket, image_size)
```

Comment:
When passing image_info from main, we can use image_info["virtual-size"], so we don't need image_size.

Since the function will be just 3 lines and used once, maybe inline it into main.